tests/webapps_tests.py

```diff
@@ -0,0 +1,82 @@
+from time import sleep
```

Comment:
I would remove this test class as it cannot be run automatically

Reply:
Talk with @Basharsh96 first :)

Reply:
I made a copy of the test and I'll add it later to the tests in the CI pipeline I'm working on. Thank you, you can safely remove it 😊